### PR TITLE
Refactor with e6

### PR DESCRIPTION
**Pull request by adeola003**
In order to complete this project, here are  the detailed steps I followed:
- I initialed the repository and started with the README and the license;
- I built the structure of the website with HTML;
- I created the main Javascript file and the CSS as well;
- I started creating different Javascript files and added some functions with the *export* prefix to import them as modules in the main;
- I added the luxon library for the date in one of the module files;
- I started implementing functions in the main Javascript file importing the modules I created;
- I added some styling and checked all the functionalities;
- I installed linters and fixed linters errors and then pushed the finished project.

### CODE REVIEW

File: modules/books.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
+const booklist = [];
```
Comment (franclobo) [inline modules/books.js:1]: - [ ] Please 🙏🏽 import the DateTime function from luxon for adding the current time to your app as required. You could check the instructions [here](https://moment.github.io/luxon/#/install?id=es6).

image

File: modules/books.js [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+export default class Book {
```
Comment (franclobo) [inline modules/books.js:2]: - **[OPTIONAL]** In order to refactor your code with ES6 notation, we could review this [article](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/class) for classes.

File: modules/books.js [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+  }
+
+  updateStorage(data) {
+    localStorage.setItem('storage-book', JSON.stringify(data));
+    this.booksList = data;
+  }
+
+  // Remove title function
+
+  removeBook(title) {
```
Comment (franclobo) [inline modules/books.js]: - [ ] It is a project requirement to refactor all functions using arrow functions, please 🙏🏽 use the correct notation, if you need help you could review this [article](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions).

File: modules/books.js [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+        this.booklist.splice(i, 1);
+        rm.updateStorage(booklist);
+        break;
+      }
+    }
+  }
+
+  // Function to display the book's list
+
+  displayBooks() {
```
Comment (franclobo) [inline modules/books.js]: - [ ] It is a project requirement to refactor all functions using arrow functions, please 🙏🏽 use the correct notation, if you need help you could review this [article](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions).

File: modules/books.js [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+      const removeBtn = bookItem.querySelector('.remove');
+      removeBtn.addEventListener('click', () => {
+        const rm = new Book();
+        rm.removeBook(book.title);
+        rm.displayBooks();
+      });
+    });
+  }
+
+  loadFromStorage() {
```
Comment (franclobo) [inline modules/books.js]: - [ ] It is a project requirement to refactor all functions using arrow functions, please 🙏🏽 use the correct notation, if you need help you could review this [article](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions).

File: modules/books.js [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+    const ld = new Book();
+    const storedBookList = localStorage.getItem('bookLists');
+    if (storedBookList) {
+      this.booklist = JSON.parse(storedBookList);
+      ld.displayBooks();
+    }
+  }
+
+  // Function to add new books
+  addBooks(ttle, athr) {
```
Comment (franclobo) [inline modules/books.js]: - [ ] It is a project requirement to refactor all functions using arrow functions, please 🙏🏽 use the correct notation, if you need help you could review this [article](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions).

**Review by franclobo — CHANGES_REQUESTED** [review]
Hi @adeola003, 

# ♻️ Required Changes ♻️

so-close-jim-halpert

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- No linter errors ✔️ 
- Correct Github flow ✔️ 
- Descriptive PR ✔️ 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by levy002 — APPROVED** [review]
## Hello @adeola003 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Status: APPROVED :fire:  :fire: 

## Highlights
- No linter errors :heavy_check_mark: 
- Well-detailed PR description and title :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**